data/strings/types_strings.txt

```diff
@@ -27209,6 +27254,7 @@
     fa = فروشگاه
     fi = Halpakauppa
     fr = Bazar
+    he = חנות כל-בו
```

Comment:
We discussed two options:
```suggestion
    he = חנות במחיר אחיד
```
Literally "shop with uniform price"
or
```suggestion
    he = חנות חיסכון
```
Litearlly "discount store", matching how it's called in iD.